### PR TITLE
doc: further align docs w/ playwright.dev

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: .github/workflows/publish_canary_docker.yml [hunk 1]
```diff
@@ -29,12 +29,12 @@ jobs:
...
-    - run: ./docs/docker/build.sh focal playwright:localbuild-focal
+    - run: ./utils/docker/build.sh bionic playwright:localbuild-bionic
+    - run: ./utils/docker/build.sh focal playwright:localbuild-focal
     - name: tag & publish
       run: |
-        ./docs/docker/tag_and_push.sh playwright:localbuild-bionic playwright.azurecr.io/public/playwright:next
-        ./docs/docker/tag_and_push.sh playwright:localbuild-bionic playwright.azurecr.io/public/playwright:next-bionic
-        ./docs/docker/tag_and_push.sh playwright:localbuild-bionic playwright.azurecr.io/public/playwright:sha-${{ github.sha }}
-        ./docs/docker/tag_and_push.sh playwright:localbuild-focal playwright.azurecr.io/public/playwright:next-focal
+        ./utils/tag_and_push.sh playwright:localbuild-bionic playwright.azurecr.io/public/playwright:next
```
Comment (dgozman) [inline .github/workflows/publish_canary_docker.yml:36]: Shouldn't this be `./utils/docker/...` ?

File: docs/out/README.md [hunk 2]
```diff
@@ -38,17 +38,18 @@ Playwright is a library to automate [Chromium](https://www.chromium.org/Home), [
    - [Page object models](./pom.md)
 1. Integrations
    - [Test runners](./test-runners.md)
-   - [Docker](./docker/README.md)
+   - [Docker](././docker.md)
```
Comment (dgozman) [inline docs/out/README.md:41]: nit: looks funny, although it does work 😄

File: docs/out/actionability.md [hunk 3]
```diff
@@ -11,13 +11,13 @@ Some actions like `page.click()` support `{force: true}` option that disable non
...
 | Actions | Performed checks |
 | ------ | ------- |
-| `check()`<br>`click()`<br>`dblclick()`<br>`tap()`<br>`uncheck()` | [Visible]<br>[Stable]<br>[Enabled]<br>[Receiving Events]<br>[Attached] |
-| `hover()` | [Visible]<br>[Stable]<br>[Receiving Events]<br>[Attached] |
-| `fill()` | [Visible]<br>[Enabled]<br>[Editable]<br>[Attached] |
-| `dispatchEvent()`<br>`focus()`<br>`press()`<br>`setInputFiles()`<br>`selectOption()`<br>`type()` | [Attached] |
-| `scrollIntoViewIfNeeded()`<br>`screenshot()` | [Visible]<br>[Stable]<br>[Attached] |
-| `selectText()` | [Visible]<br>[Attached] |
-| `getAttribute()`<br>`innerText()`<br>`innerHTML()`<br>`textContent()` | [Attached] |
+| `check()`<br></br>`click()`<br></br>`dblclick()`<br></br>`tap()`<br></br>`uncheck()` | [Visible]<br></br>[Stable]<br></br>[Enabled]<br></br>[Receiving Events]<br></br>[Attached] |
```
Comment (dgozman) [inline docs/out/actionability.md:14]: nit: `<br>` is a self-closing tag, so we might as well not add `</br>`.

File: docs/src/debug.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ a Playwright script. Developer tools help to:
 * **See console logs** during execution (or learn how to [read logs via API](./verification.md#console-logs))
 * Check **network activity** and other developer tools features
 
-<a href="https://user-images.githubusercontent.com/284612/77234134-5f21a500-6b69-11ea-92ec-1c146e1333ec.png"><img src="https://user-images.githubusercontent.com/284612/77234134-5f21a500-6b69-11ea-92ec-1c146e1333ec.png" width="500" alt="Chromium Developer Tools"></a>
+<a href="https://user-images.githubusercontent.com/284612/77234134-5f21a500-6b69-11ea-92ec-1c146e1333ec.png"><img src="https://user-images.githubusercontent.com/284612/77234134-5f21a500-6b69-11ea-92ec-1c146e1333ec.png" width="500" alt="Chromium Developer Tools"></img></a>
```
Comment (dgozman) [inline docs/src/debug.md:56]: `` is a self-closing tag as well.

**Review by dgozman — APPROVED** [review]

File: .github/workflows/publish_canary_docker.yml (hunk 1 above)
Comment (pavelfeldman) [reply .github/workflows/publish_canary_docker.yml:36]: oh.

File: docs/out/actionability.md (hunk 3 above)
Comment (pavelfeldman) [reply docs/out/actionability.md:14]: mdx requires it to be xml, not html...

File: docs/src/debug.md (hunk 4 above)
Comment (pavelfeldman) [reply docs/src/debug.md:56]: mdx requires it to be xml, not html...

**Comment by shirshak55** [comment]
@pavelfeldman Hello is it possible to browse single page ?  This link is broken https://github.com/microsoft/playwright/blob/master/docs/api.md and it was useful and every thing was in 1 page. https://playwright.dev/docs/api/class-page have very little content and searching is hard. If possible can we have one file like api.md?

**Pull request merged**